hackathon_site/dashboard/frontend/src/components/dashboard/TeamCard/__snapshots__/TeamCard.test.js.snap

```diff
@@ -3,7 +3,7 @@
 exports[`TeamCard render 1`] = `
 <DocumentFragment>
   <h2
-    class="MuiTypography-root MuiTypography-h2 MuiTypography-noWrap"
+    class="MuiTypography-root title MuiTypography-h2 MuiTypography-noWrap"
```

Comment:
This is a good example of why I don't think snapshot tests are helpful anymore, there wasn't any reason good reason this test should have failed to call out an incorrect change.